conf/docker-dcm/dcmsrv.dockerfile

```diff
@@ -1,5 +1,5 @@
 # build from repo root
-FROM centos:6
+FROM centos:8
```

Comment:
It might make more sense to leave use centos:7 here - the DCM server side stuff is tested on that OS, but I haven't tested with Cent8 yet.  I don't expect any issues, but that's different from knowing that it works.

Reply:
I reverted it back to CentOS 6. I don't want to retest DCM stuff.